src/Catalyst.Node.POA.CE.Tests/IntegrationTests/PoaConsensusTests.cs

```diff
@@ -91,6 +91,7 @@ public PoaConsensusTests(ITestOutputHelper output) : base(output)
             }
         }
 
+        //todo - Socket handlers are being disposed somewhere causing test to fail when run in CI, need to move to Synchronization so will get back to this later.
         [Fact]
         public async Task Run_ConsensusAsync()
```

Comment:
> 2020-01-27T18:14:46.6230907Z  18:14:07 [ERR] (33) SendBroadcastMessages (Catalyst.Core.Lib.P2P.IO.Messaging.Broadcast.BroadcastManager)
2020-01-27T18:14:46.6231023Z  System.NullReferenceException: Object reference not set to an instance of an object.
2020-01-27T18:14:46.6231127Z     at Catalyst.Core.Lib.IO.Transport.ClientBase.SendMessage[T](IMessageDto`1 message) in /home/vsts/work/1/s/src/Catalyst.Core.Lib/IO/Transport/ClientBase.cs:line 42
2020-01-27T18:14:46.6231272Z     at Catalyst.Core.Lib.P2P.IO.Messaging.Broadcast.BroadcastManager.SendBroadcastMessages(ProtocolMessage message, BroadcastMessage broadcastMessage) in /home/vsts/work/1/s/src/Catalyst.Core.Lib/P2P/IO/Messaging/Broadcast/BroadcastManager.cs:line 201

Reply:
Socket handlers are being disposed somewhere causing test to fail when run in CI, need to move to Synchronisation so will get back to this later, eating to much time.